index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Survey Form</title>
+	<link rel="stylesheet" href="styles.css">
```

Comment:
- [ ] I suggest you tell the student their styles are not being applied to the HTML document because they have not linked it the right way. You can go ahead and show them how to link it correctly. Is that clearer 🤔 ?